packages/contracts/src/events/maintainerrEvent.ts

```diff
@@ -2,7 +2,13 @@ export enum MaintainerrEvent {
   RuleHandler_Started = 'rule_handler.started',
   RuleHandler_Progressed = 'rule_handler.progressed',
   RuleHandler_Finished = 'rule_handler.finished',
+  RuleHandler_Failed = 'rule_handler.failed',
```

Comment:
In `\src\modules\events\events.controller.ts`

```
  @OnEvent('rule_handler.*')
  @OnEvent('collection_handler.*')
```

needs to become

```
  @OnEvent('rule_handler.started')
  @OnEvent('rule_handler.progressed')
  @OnEvent('rule_handler.finished')
  @OnEvent('collection_handler.started')
  @OnEvent('collection_handler.progressed')
  @OnEvent('collection_handler.finished')
```

as the new events are getting picked up and an error thrown